handlers/SalesforceAgentAssignedHandler.ts

```diff
@@ -41,6 +42,13 @@ export class SalesforceAgentAssigned {
 		}
 		const LAChatEndedMessage: string = await getAppSettingValue(this.read, AppSettingId.LIVEAGENT_CHAT_ENDED_MESSAGE);
 
+		const updatedRoom = await this.read.getRoomReader().getById(this.data.room.id);
+		if (updatedRoom) {
+			const salesforceAgentName = updatedRoom.customFields ? updatedRoom.customFields.salesforceAgentName : 'undefined';
+			const connectedToAgentMessage = `${ InfoLogs.CONNECTING_TO_SALESFORCE_LIVEAGENT } ${ salesforceAgentName }.`
+			await sendLCMessage(this.modify, updatedRoom, connectedToAgentMessage, this.data.agent);
+		}
+		
```

Comment:
This is not needed I guess. It is sending `Connected to salesforce agent:` message on Livechat end